gateway-server/src/main/java/org/apache/knox/gateway/services/token/impl/AliasBasedTokenStateService.java

```diff
@@ -147,4 +151,18 @@ protected void updateExpiration(final String token, long expiration) {
       log.failedToUpdateTokenExpiration(e);
     }
   }
+
+  @Override
+  protected List<String> getTokens() {
+    List<String> allAliases = new ArrayList();
+    try {
+      allAliases = aliasService.getAliasesForCluster(AliasService.NO_CLUSTER_NAME);
+      /* only get the aliases that represent tokens and extract the current list of tokens */
+      allAliases.stream().filter(a -> a.contains("TOKEN_MAX_LIFETIME_POSTFIX")).map(a -> a.substring(0, a.indexOf("TOKEN_MAX_LIFETIME_POSTFIX")))
```

Comment:
This shouldn't be a string - it should be the variable reference?

Reply:
I don't follow, this is a variable reference, I took out the string and created a variable "TOKEN_MAX_LIFETIME_POSTFIX".

Reply:
Yea you have TOKEN_MAX_LIFETIME_POSTFIX as a string with "TOKEN_MAX_LIFETIME_POSTFIX". Instead of just using TOKEN_MAX_LIFETIME_POSTFIX.